pkg/features/features.go

```diff
@@ -266,6 +266,20 @@ var GRPCRouteCoreFeatures = sets.New(
 	SupportGRPCRoute,
 )
 
+// -----------------------------------------------------------------------------
+// Features - Policy Conformance (Experimental)
+// -----------------------------------------------------------------------------
+
+const (
+	// This option indicates support for BackendTLSPolicy.
+	SupportBackendTLSPolicy SupportedFeature = "BackendTLSPolicy"
+)
+
+// ExperimentalPolicyFeatures includes all the supported features for experimental policies.
+var ExperimentalPolicyFeatures = sets.New(
```

Comment:
This set will likely be replaced by https://github.com/kubernetes-sigs/gateway-api/pull/3287, but I think it is not a sufficient reason to block this PR. In case this will be merged first, I'll adapt https://github.com/kubernetes-sigs/gateway-api/pull/3287 accordingly.

Reply:
I adapted this when rebasing for #3287.